### DISCUSSION TITLE
Does mini.fuzzy actually do anything?

### DISCUSSION THREAD

**Discussion by nDimensionalSpace** [nvim-mini/mini.nvim#256] [Q&A]
I noticed that the commands don't even seem to exist anymore . . .   Thx.

**Comment by echasnovski** [comment]
Could you elaborate on what you mean?

'mini.fuzzy' is similar in spirit to 'mini.misc': it provides functions related to fuzzy string matching. Algorithm is pretty straightforward and implementation was the fastest among Lua plugins at the time of writing it (pretty sure it still is). I personally use it for Telescope sorter, but there is also fuzzy matching for completion.
